.github/workflows/Cleanup Caches by a branch.yml

```diff
@@ -6,7 +6,7 @@
 name: Cleanup Caches by a branch
 on:
   pull_request:
-    branches: [main, 'ODS-*']
+    branches: [main, b-v*-patch*,feature-*]
```

Comment:
I think we should include the quotes, at least that's how it appears in GitHub examples:
![image](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/assets/158314644/f313194b-9aa5-42df-b4cd-9cc16ab33ddc)